lib/core/src/sdk.rs

```diff
@@ -788,11 +788,54 @@ impl LiquidSdk {
                 };
                 payment_destination = SendDestination::Bolt11 { invoice };
             }
-            _ => {
+            Ok(_) => {
                 return Err(PaymentError::Generic {
                     err: "Destination is not valid".to_string(),
                 });
             }
+            Err(_) => {
```

Comment:
I think we must add the bolt12 & offer to the input parser. I don't see how would the user app recognize it is a bolt12/offer otherwise.

Reply:
Done.